docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,97 @@
+for all your **LCA1** accounts. We add MFA to the Root User.  We securely store and rotate the password.
+We have alarms set up to detect any usage of this important and powerful admin user.  Our internal
+procedures ensure that the root user is only used for approved use, requires approval, and is only
+used for very rare cases.  For a list of things that only the root user can do,
+[see this page](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#root-user-tasks).
+In the rare event that any of these actions are needed, [rc-support@colorado.edu](mailto:rc-support@colorado)
+can assist with submitting a ticket with the CFS team.
+
+**Centralized Audit Logging**
+
+When using the CFS **LCA1** landing zone all user, role and service actions are centrally recorded
+and stored with the proper data retention and audit policies.
+This includes actions in the AWS Console, the AWS CLI, and AWS SDKs and APIs.
+For more information, see [AWS CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html) documentation.
+This feature allows you to focus on your application logging without the need to build and maintain an extra audit logging layer.
+
+**Managed Network and Firewall**
+
+When using the CFS managed [AWS VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html),
+you have a secure encrypted private subnet connecting back to CU Boulder campus.
+You also get a internet subnet that we manage.  Both subnets are inspected by a firewall.
+Both inbound and outbound traffic are inspected.
+The firewall and campus connection are managed by a partnership with
+[OIT Network and Internet Services](https://oit.colorado.edu/services/network-internet-services).
+
+**Training and Support**
+
+The OIT [Research Computing Support Team](https://curc.readthedocs.io/en/latest/#meet-the-rc-user-support-team)
+is eager and ready to assist with training for you and your team in public cloud usage.
+RC User Support also provides support services to help give advice and troubleshoot various cloud services.
+RC User Support can also help with billing and cost optimization.
+
+Not sure where to start with your cloud journey?  RC User Support is here to guide and assist!
+Reach out to us at [rc-help@colorado.edu](mailto:rc-help@colorado.edu).
+
+### SSP Quickstart Process
+1. Determine your [Data Classification](https://www.cu.edu/data-governance/resources-support/data-classification).
+2. Reach out to [rc-help@colorado.edu](mailto:rc-help@colorado.edu) so that we can gather basic questionnaire information about your workload.  This will include your data classification.  We can assist with your data classification if you are unsure.
+3. Read and understand the [Shared Responsibility Model](../getting-started/shared-responsibility-model.md).
+4. Work with [OIT IT Security](https://oit.colorado.edu/services/it-security) to get your SSP submitted and reviewed.
```

Comment:
I'm not sure if this is the time, but the process for doing and reviewing and SSP is still pretty poorly documented. Are we expanding on that here or asking OIT Security to write up a page on the website? If I were a customer, I'd be lost at this point.

Reply:
Agree 100%, we have nothing to point to.  This is one of the major gaps identified as part of the story AC.  If you have suggestions to expand, please let me know so we can discuss.  I'm not comfortable speculating on the process workflow that OIT-Sec is going to require from our users.  I'm not sure they even know.

Reply:
No change unless you have something we should add.